apps/frontend/package.json

```diff
@@ -9,19 +9,26 @@
     "lint": "eslint . --ext ts,tsx --report-unused-disable-directives --max-warnings 0",
     "format": "prettier --write ./src",
     "format:ci": "prettier --check ./src",
-    "test": "vitest",
+    "test": "cypress run --component",
     "preview": "vite preview"
   },
   "dependencies": {
+    "@emotion/react": "^11.11.4",
+    "@emotion/styled": "^11.11.5",
+    "@mui/material": "^5.15.19",
+    "axios": "^1.7.2",
     "react": "^18.2.0",
-    "react-dom": "^18.2.0"
+    "react-dom": "^18.2.0",
+    "react-router": "^6.23.1"
```

Comment:
nit: any explicit reason to add these deps in with the Cypress PR? I know you've mentioned that you'll be using them explicitly but it's good practice to only add a dep once it's required (let's say we have to revert this PR, we don't also revert the added deps that other parts of the code require).

Reply:
We should be using all of this stuff nearly immediately, there's a bunch of config that needed to be handled for getting cypress to have code coverage metrics and for working with Vite. Since we're doing TDD we're going to need cypress for any additions we're doing.